### PR TITLE
add basic Docker-isation

### PR DESCRIPTION
**Pull request by rjw57**
Add a simple Dockerfile for the smswebapp which can be used to host the app. Take the opportunity to rationalise the requirements files into a single directory to avoid proliferation as outlined in uisautomation/lookupproxy#10.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,40 @@
+FROM python:3.6-alpine
```
Comment (abrahammartin) [inline Dockerfile]: should we create a docker-compose for the production Dockerfile as well?

Comment (rjw57) [reply Dockerfile]: Good idea. Inspired by the [standard cookiecutter Django template](https://github.com/pydanny/cookiecutter-django/tree/master/%7B%7Bcookiecutter.project_slug%7D%7D) I think this is the threshold where we split them into separate files.

**Comment by rjw57** [comment]
OK, split the docker-compose and took the opportunity to improve the documentation around the docker-compose stuff. If this gets merged, I'll teleport it to django-boilerplate.

File: compose/development.Dockerfile [hunk 2]
```diff
@@ -1,10 +1,14 @@
-FROM python:3.6
+FROM python:3.6-alpine
```
[the remark this answers is not in the document]
Comment (rjw57) [reply compose/development.Dockerfile]: I've no strong opinion on this so I'm happy to make this ``FROM python:3.6``.

**Comment by rjw57** [comment]
@abrahammartin Code review done

File: compose/tox.yml [hunk 3]
```diff
@@ -8,7 +8,7 @@ services:
   tox:
     build:
       context: ..
-      dockerfile: ./Dockerfile
+      dockerfile: ./compose/development.Dockerfile
```
Comment (abrahammartin) [inline compose/tox.yml]: isn't it better to execute tox using the production enviroment (alpine) rather that the dev enviorment? I was only thinking on the dev enviroment (debian) for devserver / debug puproses

Comment (rjw57) [reply compose/tox.yml]: Ah, OK. Yes, we can do that. Will fix.

Comment (rjw57) [reply compose/tox.yml]: And, done in ea2a759

**Comment by rjw57** [comment]
Merged after @abrahammartin indicated at this morning's standup that it can be moved to "done".

**Pull request merged**